### PR TITLE
Add pyirf in setup.py

### PR DESCRIPTION
**Pull request by vuillaut**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ def find_scripts(script_dir, prefix):
         'numba',
         'numpy',
         'pandas',
+        'pyirf~=0.2.0',
```
Comment (maxnoe) [inline setup.py:56]: We just released 0.3

Comment (vuillaut) [reply setup.py:56]: Are the things introduced previously here compatible?

Comment (maxnoe) [reply setup.py:56]: There is already pyirf stuff in the master?

Comment (vuillaut) [reply setup.py:56]: I mean I just wanted to solve the install issue but yeah I can bump to 0.3.0 at the same time

Comment (maxnoe) [reply setup.py:56]: Ah, yes the reading part. I had a quick glimpse and I think nothing with respect to that changed in 0.3

File: environment.yml [hunk 2]
```diff
@@ -18,4 +18,4 @@ dependencies:
       - pytest-ordering
       - git+git://github.com/cta-observatory/ctapipe_io_lst@v0.5.3
       - ctaplot
-      - pyirf~=0.3.0
+      - pyirf=0.3.*
```
Comment (maxnoe) [inline environment.yml:21]: Shouldn't anaconda just pass this to pip and it works?

Comment (vuillaut) [reply environment.yml]: I thought so too and applied the same semantic but it was not recognized...

**Comment by vuillaut** [comment]
Tests will pass when pyirf v0.3.0 is deployed on pypi.

**Comment by maxnoe** [comment]
It is deployed now, sorry we hat an error in the travis config that prevented the automatic upload

**Review by maxnoe — APPROVED** [review]

**Pull request merged**